packages/react-native-device-activity/src/index.ts

```
@@ -307,20 +358,25 @@
+}
+
+export function unblockSelectedApps(
+  familyActivitySelectionId: string,
   triggeredBy?: string,
 ): void {
-  return ReactNativeDeviceActivityModule?.blockApps(
-    familyActivitySelectionStr,
+  return ReactNativeDeviceActivityModule?.unblockSelectedApps(
+    familyActivitySelectionId,
     triggeredBy,
   );
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)